src/main/java/io/cdap/plugin/snowflake/common/client/SnowflakeAccessor.java

```diff
@@ -74,6 +75,38 @@ public void runSQL(String query) {
     }
   }
 
+  /**
+   * Returns field descriptors for specified table name
+   * @param schemaName The name of schema containing the table
+   * @param tableName The name of table whose metadata needs to be retrieved
+   * @return list of field descriptors
+   * @throws SQLException If an error occurs while retrieving metadata from the database
+   */
+  public List<SnowflakeFieldDescriptor> describeTable(String schemaName, String tableName) throws SQLException {
```

Comment:
please keep the method name to reflect what exactly is being done, something like `getFieldDescriptors`

Reply:
Renamed dad7ba90da44181fa5a237eba945ace1218382cf